src/R/Editor/Impl/Selection/SelectWordCommand.cs

```diff
@@ -24,60 +22,13 @@ public sealed class SelectWordCommand : ViewCommand {
         public override CommandResult Invoke(Guid group, int id, object inputArg, ref object outputArg) {
```

Comment:
Right, but we're providing our own navigator now. So do we still need to override that command, or can we just let the default implementation of the command do its thing (and use our navigator)?